### PR TITLE
Asp.net/task 1

### PR DESCRIPTION
**Pull request by gecmenow**

### CODE REVIEW

File: ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31613.86
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "KramDeliveryFoodAPI", "KramDeliveryFoodAPI\KramDeliveryFoodAPI.csproj", "{DAC59FC3-BA6C-4B95-87AA-EC378B9BAC0F}"
```
Comment (yevhenbilushko) [inline ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln:6]: please add build check in GitHub

File: ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI/Controllers/ProductController.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+using System.Collections.Generic;
+
+namespace KramDeliveryFoodAPI.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ProductController : ControllerBase
+    {
+        [HttpGet]
+        public IList<KramDeliverFoodCompleted.Models.Product> Get()
```
Comment (yevhenbilushko) [inline ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI/Controllers/ProductController.cs:13]: remove "KramDeliverFoodCompleted.Models." and use using to link that type

File: ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
```
Comment (yevhenbilushko) [inline ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI/Startup.cs:25]: remove comments

File: KramDeliveryFood_v3/KramDelivery.Domain/Services/ProductService.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        }
+
+        public void UpdateProduct(Product product)
+        {
+            _context.Products.Update(product);
+        }
+
+        public Product GetProductById(Guid id)
+        {
+            throw new NotImplementedException();
```
Comment (yevhenbilushko) [inline KramDeliveryFood_v3/KramDelivery.Domain/Services/ProductService.cs:41]: implement it or remove methods

File: KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs [hunk 5]
```diff
@@ -1,12 +1,68 @@
-﻿using System;
+﻿using KramDelivery.Domain.Service;
+using System;
+using System.Linq;
 
 namespace KramDeliveryFood_v3
 {
     public class Program
```
Comment (yevhenbilushko) [inline KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs:7]: i think we dont need this project here

File: ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln (hunk 1 above)
Comment (yevhenbilushko) [reply ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln:6]: and close PRs from previous modules

Comment (gecmenow) [reply ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln:6]: Even if I didn't get approve there?

File: KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs (hunk 5 above)
Comment (gecmenow) [reply KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs:7]: if we don't need this project, so should I create another project for business logic, or just create a folder in API project?

File: ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln (hunk 1 above)
Comment (yevhenbilushko) [reply ASP.NET/KramDeliveryFoodAPI/KramDeliveryFoodAPI.sln:6]: yes

File: KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs (hunk 5 above)
Comment (yevhenbilushko) [reply KramDeliveryFood_v3/KramDeliveryFood_v3/Program.cs:7]: do not use "console app" project type 
move to "class library" project type

**Comment by yevhenbilushko** [comment]
switch to class lib proj and merge

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**